database/migrations/2023_02_27_092130_add_scope_locations_setting.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddScopeLocationsSetting extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('settings', function (Blueprint $table) {
+            $table->boolean('scope_locations_fmcs')->default('0');
```

Comment:
This is a nitpick but can you group this column with `full_multiple_companies_support` please?
```suggestion
            $table->boolean('scope_locations_fmcs')->default('0')->after('full_multiple_companies_support');
```

Reply:
Ok, changed.